system-parachains/bridge-hubs/bridge-hub-kusama/src/lib.rs

```diff
@@ -295,7 +296,7 @@ parameter_types! {
 impl pallet_transaction_payment::Config for Runtime {
 	type RuntimeEvent = RuntimeEvent;
 	type OnChargeTransaction =
-		pallet_transaction_payment::CurrencyAdapter<Balances, DealWithFees<Runtime>>;
+		pallet_transaction_payment::FungibleAdapter<Balances, ResolveTo<StakingPot, Balances>>;
```

Comment:
This PR changes the tx fees for some of the system parachains from `DealWithFees` (80% treasury, 20% block author) to `StakingPot` (100% goes to collators iiuc).

First, I want to raise visibility of this to make sure it's what we want (I haven't seen any convos on the topic so I don't know what's the general consensus).
Second, let's make it visible in the CHANGELOG.

Reply:
No I wanna only change it from currency to fungible. parachains_common::DealWithFees deposits everyting to staking pot.
https://github.com/paritytech/polkadot-sdk/blob/89b67bc69e5b8cdad49c378f6db0a48873548d35/cumulus/parachains/common/src/impls.rs#L62

Reply:
Yes, they have a similar name, it can be confusing.